index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>form</title>
+</head>
+<body>
+    <form action="my handling form page" method="post"></form>
+    <h1>
+        assignment:
+    </h1>
+    <fieldset>
+         <legend>personal details></legend>
+            <label for="salutation">
+                <span>salutation</span>
+            </label>
+            <br>
+               <ul>
+                <li>
+                    <label for="first name">First name:</label>
+                    <input type="text" id=" name" name="last name">
+                </li>
+                <br>
+                                <li>
+                    <label for="name">Last name</label>
```

Comment:
| Original Desing  |
| --------------------- | 
![examplee](https://github.com/broad65/form/assets/61404403/2d0efdee-47b1-4fda-be03-624714c31b4d)
  
| Your Desing |
|---------------- |
![Screenshot from 2023-07-26 00-31-57](https://github.com/broad65/form/assets/61404403/3be89ce6-7797-4110-9c43-f659ca8394e4)
 |

Your design looks good but a few things to consider to make it better, there is something we call `sticking to the design`. Provided a design/wireframe in this case is the assignment screenshot, try as much as possible to stick to the provided design.

1. Tittle.
The original design has no title as compared to your design.
![Screenshot from 2023-07-26 00-31-57](https://github.com/broad65/form/assets/61404403/fa77697b-0ae2-4749-afea-5750a23e821a)

2. Salutation.
Original Design contains a drop-down that allows the user to select the salutation which also lacks in yours.
![example0](https://github.com/broad65/form/assets/61404403/ef606cb7-6d2d-49e2-b833-8ed724ba576d)

3. Email & Gender
Please add the Gender using  [radio button](https://www.w3schools.com/tags/att_input_type_radio.asp)  and [email](https://www.w3schools.com/tags/att_input_type_email.asp)

### 🔴 N/B Observe the clickable links for Hints